### PR TITLE
Add workload over-view documentation

### PR DESCRIPTION
**Pull request by CannonLock**

### CODE REVIEW

**Comment by CannonLock** [comment]
You can view this locally at http://0.0.0.0:8011/documentation/overview/references/workload-overview/#machine-learning. 

To do this checkout this branch and run:

```
docker run -it -p 8011:8011 -v ${PWD}:/docs/user-documentation hub.opensciencegrid.org/opensciencegrid/path-portal-documentation:latest
```

**Comment by ChristinaLK** [comment]
Cannon, I couldn't get the preview to work. 😢 When I start the container, it says it's serving on port 8000, but neither port 8000 or 8011 work to preview .

File: documentation/PATh/overview/references/workload-overview.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+The size of the file in gigabytes
+
+## Examples
+
+To help provide an idea of how you might update your existing workload to the HTC model we have provided an example below. 
+
+!!! tip
+
+      The PATh team has alot of experience updating workloads to fit the HTC environment. Please contact support if you
```
Comment (ChristinaLK) [inline documentation/PATh/overview/references/workload-overview.md:85]:
```suggestion
      The PATh team has significant experience updating workloads to fit the HTC environment. Please contact support if you
```

**Review by ChristinaLK — COMMENTED** [review]
To summarize Christina's thoughts: smallest unit is a task, make the boxes in the diagrams flow in one direction. :)

**Comment by rachellombardi** [comment]
@CannonLock do you want me to review this? If yes, please add me as a reviewer. Thanks!

**Pull request closed without merging**